enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -101,16 +101,23 @@
       {:session session, :redirect-url redirect-url, :jwt-data jwt-data})))
 
 (defn- check-jwt-enabled []
-  (api/check (sso-settings/jwt-enabled)
-             [400 (tru "JWT SSO has not been enabled and/or configured")]))
+  (when-not (sso-settings/jwt-enabled)
+    (throw
+     (ex-info (tru "JWT SSO has not been enabled and/or configured")
+              {:status "error-sso-jwt-disabled" :status-code 402}))))
```

Comment:
If it would help us give better guidance, we could differentiate between not enabled and not configured by including the value at `(sso-settings/jwt-enabled)` in this response.

Reply:
oh that would be really nice actually. let me do that